well_records.rb

```diff
@@ -0,0 +1,183 @@
+require 'open-uri'
+
+module Importer
+  module Oklahoma
+    class WellRecords
+      attr_reader :starts_at, :ends_at
+
+      HOST = "http://imaging.occeweb.com/"
+      OGWR_PATH = "imaging/OGWellRecords.aspx"
+      AWS_BUCKET_PATH = "well_data/WY/scraped"
```

Comment:
I realize I didn't include the details for this. the path should be "well_data/OK/scraped"